tasmota/tasmota_xsns_sensor/xsns_18_pms5003.ino

```diff
@@ -370,20 +441,42 @@ void PmsShow(bool json) {
 #ifdef PMS_MODEL_PMS5003T
     float temperature = ConvertTemp(pms_data.temperature10x/10.0);
     float humidity = ConvertHumidity(pms_data.humidity10x/10.0);
+    int epa_us_aqi;
+    // When in Fahrenheit include US AQI
+    if (Settings->flag.temperature_conversion) {
+      epa_us_aqi = compute_us_aqi(usaEpaStandardPm2d5Adjustment(pms_data.pm25_standard, humidity));
+    }
 #endif // PMS_MODEL_PMS5003T
+    int us_aqi;
+    const char* european_aqi;
+    // Use US AQI for Fahrenheit, EAQI (European Air Quality Index) for Celsius
+    if (Settings->flag.temperature_conversion) {
+      us_aqi = compute_us_aqi(pms_data.pm25_standard);
+    } else {
+      european_aqi = compute_european_aqi(pms_data.pm25_standard);
+    }
 
     if (json) {
```

Comment:
This section of code was inconsistent with trailing commas.  I changed it so each new json string we append starts with a comma and does not have a trailing comma